src/modules/ordersTable/pure/OrdersTableContainer/OrdersTable.tsx

```diff
@@ -435,12 +450,15 @@ export function OrdersTable({
                   isRowSelected={!!selectedOrdersMap[order.id]}
                   isOpenOrdersTab={isOpenOrdersTab}
                   order={order}
+                  twapOrderParts={composableCowOrders[order.id] ? partsOrders[order.id]?.length || 0 : undefined}
```

Comment:
non trivial statements should be created above as a `const twapOrderParts = ...` so we don't complicate the markup